tests/material_bsdf_test.py

```diff
@@ -61,3 +62,10 @@ def test_to_and_from_dict():
     with open(bsdf_in.bsdf_file, 'r') as f1, \
             open(bsdf_from_dict.bsdf_file, 'r') as f2:
         assert f1.read() == f2.read()
+
+
+def test_bsdf_to_json():
+    """Ensure that the BSDF dictionary is serialize-able to JSON."""
+    test_bsdf = BSDF(klems_bsdf_file)
+    bsdf_dict = test_bsdf.to_dict()
+    bsdf_str = json.dumps(bsdf_dict)
```

Comment:
Do you need to check something for the translation to ensure it has been successful?

Reply:
Sure. The `from_dict` test above this kinda already checks that part but I guess it doesn't hurt to do it again here.